src/test/run-rbd-tests

```diff
@@ -2,13 +2,18 @@
 
 # this should be run from the src directory in the ceph.git
 
-CEPH_SRC=$(pwd)
-export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CEPH_SRC/.libs"
+source $(dirname $0)/detect-build-env-vars.sh
+if [ -e CMakeCache.txt ]; then
+    CYTHON_MODULES_DIR=$CEPH_LIB/cython_modules
+else
+    CYTHON_MODULES_DIR=$CEPH_SRC/build
+fi
+
+CEPH_SRC=$CEPH_ROOT/src
```

Comment:
@tchaikov Ah, doing the last moment cleanup I wrongly moved this line below CYTHON_MODULES_DIR definition. I have created a fixup: https://github.com/ceph/ceph/pull/10134